### ISSUE TITLE
Add RenderTreeBuilder samples

### ISSUE THREAD

**Issue by dodyg** [dodyg/practical-aspnetcore#249]

**Comment by lohithgn** [comment]
is this for Blazor ?

**Comment by dodyg** [comment]
Yup. https://docs.microsoft.com/en-us/aspnet/core/blazor/advanced-scenarios?view=aspnetcore-5.0

I haven't never used it in real life but it's a good thing to learn.
